packages/landing/src/components/debug.tsx

```diff
@@ -423,7 +424,9 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
 
   getSourcesIds(type: string): string[] {
     const style = this.props.map.getStyle();
-    return Object.keys(style.sources).filter((id) => id.startsWith('basemaps') && style.sources[id].type === type);
+    return Object.keys(style.sources).filter(
+      (id) => (id.startsWith('basemaps') || id.startsWith('LINZ')) && style.sources[id].type === type,
```

Comment:
I think we should do a exclude of the `__hillshade` rather than the include? as its only the duplicated hillshade layer we are trying to remove.

Reply:
This is only filter the useful ones which might startwith `basemaps-` or `LINZ-`

Reply:
We don't want others to be added to layer dropdown like `oms` source

Reply:
what source is oms ? I could see us in the future removing the forced `basemaps-` prefix.

Reply:
![image](https://github.com/linz/basemaps/assets/12163920/6efbfea5-b6c0-4d39-b596-d4a2c6a68e5d)
The layer slider will add open street map(`osm`) source and aerial source into the style, which might added to the layer dropdown if we don't filter them out.

Reply:
These wouldn't be `raster-dem` sources though would they? I wonder if we should type the `type` ? or doc this function more

Reply:
This function been used for Layer drop down as well. So the osm and aerial will showing up in the Layer drop down.

Reply:
Hope that explained the problem for not filtering `basemaps-` and `LINZ-`
![image](https://github.com/linz/basemaps/assets/12163920/bb18f2e2-5bfd-4eac-ae53-46b123470850)